pytorch_toolkit/ote/ote/modules/arg_converters/base.py

```diff
@@ -202,28 +200,25 @@ def _convert_args_by_hooks_for_action(hooks_for_action, model_template_path, arg
         extra_args = hooks_for_action.get_extra_args_hook(args)
         update_args.update(extra_args)
 
-        template_folder = os.path.dirname(model_template_path)
         converted_args = {
-            'config': os.path.join(template_folder, args['config']),
+            'config': args['config'],
```

Comment:
Ilya, I'm sorry, but I do not understand how it is possible to skip the `template_folder` here
It is great if you managed to avoid this dependency, but I reviewed the changes and did not find where do you move 
`os.path.join(template_folder, ...)`

Or you force absolute paths here?

Reply:
I did not found when it appeared (usage of template folder), but it is assumed that all scripts are run from template folder, so that there is no need to pass template folder.

Reply:
I see. OK.
(Although I like absolute paths a bit more -- this is more reliable. But in this concrete case it should be ok.)